lib/src/dsl_util.rs

```diff
@@ -423,16 +423,28 @@ impl<R: RuleType> StringLiteralParser<R> {
 }
 
 /// Map of symbol and function aliases.
-#[derive(Clone, Debug, Default)]
-pub struct AliasesMap<P> {
-    symbol_aliases: HashMap<String, String>,
+#[derive(Clone, Debug)]
+pub struct AliasesMap<P, V = String> {
```

Comment:
nit: remove default type or use it consistently

(I prefer not adding `V = String`.)